index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
+            <img class="menubars" src="menu.png" alt="Menu"> 
+        </nav>
+        <section class="homepage">
+            <div class="intro">
+                <h2>
+                    I'm Abu Talha. <br>  
+                    Glad to see <br> you!
+                </h2>                                  
```

Comment:
-  [ ] For accessibility to your page, Kindly make sure you use `h1` for the heading tag `h1` for that reason google (and other search engines) can know about a specific topic on your web page, in this case, you.